src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
+   * users click the image or drag another image on top of it,
+   * much like the `sky-file-drop` component in the
+   * [file attachment module](https://developer.blackbaud.com/skyux/components/fileattachments).
+   */
   @Input()
-  public set src(value: SkyAvatarSrc) {
-    this._src = value;
+  public set canChange(value: boolean) {
+    this._canChange = value;
   }
 
-  public get name(): string {
-    return this._name;
+  public get canChange(): boolean {
+    return this._canChange;
   }
 
+  /**
+   * Specifies the name of the record that the avatar represents.
+   * If the `src` property does not specify an image, the component extracts initials
+   * from the first and last words of this name and displays them in place of the missing image.
+   * To ensure that the component extracts the correct initials, specify a name with no prefix
+   * or suffix. You can specify just the initials with a space between them.
+   */
   @Input()
   public set name(value: string) {
     this._name = value;
   }
 
-  public get canChange(): boolean {
-    return this._canChange;
+  public get name(): string {
+    return this._name;
   }
 
+  /**
+   * Specifies an image to display to identify a record.
+   */
   @Input()
-  public set canChange(value: boolean) {
-    this._canChange = value;
+  public set src(value: SkyAvatarSrc) {
+    this._src = value;
   }
 
+  public get src(): SkyAvatarSrc {
+    return this._src;
+  }
+
+  /**
+   * Specifies the maximum file size for the image in bytes.
+   */
   @Input()
   public maxFileSize = 500000;
 
+  /**
+   * Emits [a `SkyFileItem` object](https://developer.blackbaud.com/skyux/components/avatar#skyfileitem-properties)
```

Comment:
This link points to a section in the existing docs that is now missing. Do we still need the `SkyFileItem` properties described in https://developer.blackbaud.com/skyux/components/avatar#skyfileitem-properties? That might be complicated because digging into the code, it looks like we import those properties into this module from the `forms` repo. Any idea what we need to do here?

Reply:
Just realized that you already had a comment in this PR about this, so I'm resolving this comment.